lib/tlog/es_json_reader.c

```diff
@@ -128,7 +128,7 @@ tlog_es_json_reader_format_url_pfx(char **purl_pfx,
 #undef ESC
 
     if (asprintf(&url_pfx,
-                 "%s?q=%s&fields=&_source&sort=%s&size=%zu&from=",
```

Comment:
Hmm, was support for `_source` removed as well? Why do we need to remove it?

Reply:
Ah, right. Was a bit overzealous with the removal there. Since we aren't specifying any value there, is there any reason to keep `_source` in the query string?

Reply:
I have no idea if it's even used by Elasticsearch. I can't find any documentation about it and don't remember why I added it there. I'll need to do a little testing on this. However, in any case, removing `_source` needs to be a separate commit (within this PR is OK), if you could do it.

Reply:
There seems to be no difference in Elasticsearch 2.4.6 response, at least, so I'll have to assume that these particular URL parameters came from some early prototyping code, where I didn't care much. Let's remove `_source` as well. Could you add a separate commit to this PR, something like "Remove unused _source parameter from ES query URL"?